src/Akka.Persistence.Hosting.Tests/UnifiedApiSpecs.cs

```diff
@@ -0,0 +1,322 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Journal;
+using Akka.Util;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.Hosting.Tests;
+
+/// <summary>
+/// Shared test resources for unified API specs
+/// </summary>
+public static class UnifiedApiTestResources
```

Comment:
Tests the "unified" Akka.Persistence.Hosting APIs and tries to get test coverage over the `PersistenceHostingExtensions`